tests/test_constants.py

```diff
@@ -0,0 +1,41 @@
+import collections
+
+from eth_utils import (
+        keccak,
+    )
+
+from rlp.codec import encode_raw
+
+from trie.constants import (
+        BLANK_NODE,
+        BLANK_HASH,
+        BLANK_NODE_HASH,
+    )
+
+from trie.smt import SparseMerkleTree as SMT
+
+
+def test_hash_constants():
+    assert BLANK_HASH == keccak(BLANK_NODE)
+    assert BLANK_NODE_HASH == keccak(encode_raw(b''))
+
+
+def test_smt256_empty_hashes():
+    DEPTH = 256  # Default depth is 32 bytes
+
+    # Start at the bottom
+    EMPTY_LEAF_NODE_HASH = BLANK_HASH
+    EMPTY_NODE_HASHES = collections.deque([EMPTY_LEAF_NODE_HASH])
+
+    # More hashes the lower you go down the tree (to the root)
+    # NOTE: Did this with different code as a sanity check
+    for _ in range(DEPTH - 1):
+        EMPTY_NODE_HASHES.appendleft(keccak(EMPTY_NODE_HASHES[0] + EMPTY_NODE_HASHES[0]))
+    EMPTY_ROOT_HASH = keccak(EMPTY_NODE_HASHES[0] + EMPTY_NODE_HASHES[0])
```

Comment:
So, the root would change with different choices of the default. I'm not sure a utility would be a good suggestion, but I'm open to that. I wanted to calculate it here in a different way so that that test does not depend on the implementation. To put the branch explicitly would be almost 260 hardcoded lines, so I though this was a good compromise to doing that.

Also, we have the `smt.calc_root(k, v, b)` utility that can obtain a root hash.

Reply:
So, the root would change with different choices of the default. I'm not sure a utility would be a good suggestion, but I'm open to that. I wanted to calculate it here in a different way so that that test does not depend on the implementation. To put the branch explicitly would be almost 260 hardcoded lines, so I though this was a good compromise to doing that.

Also, we have the `smt.calc_root(k, v, b)` utility that can obtain a root hash.